lib/ueberauth/strategy/edm.ex

```diff
@@ -20,6 +20,10 @@ defmodule Ueberauth.Strategy.EDM do
     Logger.debug "These are the requested scopes: " <> scopes
     opts = [ scope: scopes ]
     opts = if conn.params["state"], do: Keyword.put(opts, :state, conn.params["state"]), else: opts
+    opts =
```

Comment:
I'm ok with that for now.
If we have to deal with another option in the future I would like to see something like:

```elixir
  @whitelist_options ["state", "organization_domain"]
  
  def handle_request!(conn) do
    # ...
    opts = keep_whitelist_options(conn, opts, @whitelist_options)
  # ...
  end

  def keep_whitelist_options(conn, opts, [key | whitelist]) do
    opts =
      if conn.params[key],
        do: Keyword.put(opts, String.to_atom(key), conn.params[key]),
        else: opts

    keep_whitelist_options(conn, opts, whitelist)
  end

  def keep_whitelist_options(conn, opts, []), do: opts
```